main.cpp

```diff
@@ -136,6 +138,11 @@ int Mp3Player::findFiles()
     return count;
 }
 
+void Mp3Player::onOffSwitchColorCout()
```

Comment:
Name things what they are. 
Name functions based on what they do, and make sure that the name includes a verb
- [ ] revise